src/opera/value.py

```diff
@@ -48,16 +48,41 @@ def eval(self, instance, key):
             raise DataError("Cannot use an unset value: {}".format(key))
 
         if self.is_function:
-            return self.eval_function(instance)
+            return Value.eval_function(self.data, instance)
+
+        if isinstance(self.data, dict):
+            result_map = {}
+            for map_key, value in self.data.items():
+                result_map[map_key] = Value.check_eval_function(value,
+                                                                instance)
```

Comment:
This can be one-liner.

```suggestion
                result_map[map_key] = Value.check_eval_function(value, instance)
```

Reply:
@anzoman IDK linter treats it as an error as line length exceeds 79 chars

Reply:
We've increased the line length limit to 120, so this isn't an issue any more.